domains/dinokhoa.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "n0td1n0kh0a",
+           "email": "dinokhoa@proton.me",
+           "discord": "888370186003050526",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.qx_BgIxje-NPHNb6QDw9dXHOstJKAnAYRL5AKcfrDCCUDT8tzK06v1fIRIw1KXhnIKl66nHlw4lLQz3ZeZYRnLdCOslkLmq9p1GAXk5566eppX11HJe7I40ERbckiXbs0Nla1xCHqYbEXAmkXgTi8y9uNS-JnVXiT9ECaZu7ay0UTo9NA6JGcmY5Hvc_WU-flnNtmkr_5EFE0SZKZnPkQtv4xOGzvIBFpUm1Xsv4lXeiKLp93qmh_42eG_vU4P405_AmgxgziDirkLU1itg-mif074bCcqf16jmJsMgQ7WBslxrlIRoC1ukHowA_rjaoyR11woueRNjDEJDGJpV_LQ.9KTWGAKMM-I9HwYyaPJZAg.GXEfjxPeCdC1PrJcd1AgHvkk2572vMcggMdsQRgmgs3v2BxQP6zhgQBEVpw5S5E6bvo9rxSwHgE55cqlMDnhj6_i4FlFhBDKC44Xg2L6x0M._7SnVuj-NZ0E8EHM-ZRRig"
+        },
+    
+        "record": {
+            "CNAME": "52.221.227.30"
```

Comment:
CNAME records should be hostnames not IPs. You might have been mistaken for A records. Please check the docs for correct records: https://docs.is-a.dev